pkgs/tools/filesystems/dbxfs/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, fetchgit, python3Packages, fusePackages }:
+
+let
+  block-tracing = python3Packages.buildPythonPackage rec {
+    pname = "block_tracing";
+    version = "1.0.1";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
```

Comment:
```suggestion
      inherit pname version;
```

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/155134/commits/6b30949169f06715eb8973b6ddb13b19467be98a